DigitalSignageUAP/SlideshowPage.xaml.cs

```diff
@@ -307,8 +307,20 @@ void PlayAudio()
                             DisplayObject DO = new DisplayObject();
                             string filename = fileElement.Attribute("path").Value.Substring(fileElement.Attribute("path").Value.LastIndexOf('/') + 1);
                             StorageFile file = await tmp.CreateFileAsync(filename, CreationCollisionOption.ReplaceExisting);
-                            byte[] bytes = File.ReadAllBytes(filename);
-                            await FileIO.WriteBufferAsync(file, WindowsRuntimeBufferExtensions.AsBuffer(bytes));
+                            Uri fileElementUri;
+
+                            if (Uri.TryCreate(fileElement.Attribute("path").Value, UriKind.Absolute, out fileElementUri)
```

Comment:
looks good, but I have one question: I thought there was a regression somehow and reading files from a web location was broken... but from the PR it looks it never worked... is that correct?